vb-fe/src/components/loading/Loading.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FunctionComponent } from "react";
+
+import { Modal, ModalOverlay, ModalContent, ModalHeader, ModalBody, Icon } from "@chakra-ui/react";
+
+import { ReactComponent as LoadingIcon } from "assets/icons/loading.svg";
+
+import "components/loading/Loading.scss";
+
+interface Props {
+  show: boolean;
+}
+
+export const Loading: FunctionComponent<Props> = ({ show }) => {
+  const onClose = () => {
+    console.log("Loading closed");
```

Comment:
yeah i left it as a placeholder and forgot to update it.

the thing is when this onclose happens the loading in the state will be false because thats the way it hides

Reply:
but chakra requires an `onClose` method to be preset :(

Reply:
Oh lol, we can just do like `() => null` or something.  I'm really not sure what else we could utilize the `onClose` for right now

Reply:
thats also not allowed LOL